### DISCUSSION TITLE
403 and 429 when viewing images on Frigate+

### DISCUSSION THREAD

**Discussion by Crosenhain** [blakeblackshear/frigate#8584] [Ask A Question] [answered]
For some reason when I've uploaded an image from Frigate to Frigate+, the images don't display and the web browser console says HTTP code 429 for the ingest.sentry.io URL, and 403 for the image thumbnail URL. I tried deleting and uploading new images but this didn't change the outcome. Manually uploading an image has the same "403 Forbidden" result, and no image is uploaded. 

I have had a Frigate+ account for a while but only today subscribed to the paid subscription. Before I upgraded the old images I had uploaded weren't showing up either. 

![image](https://github.com/blakeblackshear/frigate/assets/1342284/3cc33013-a56d-4049-b336-7200c6bdcde4)

**Comment by blakeblackshear** [comment, marked as answer]
There are few situations where this happens when cookies get messed up. I would try clearing your cookies for all frigate.video domains, especially images.frigate.video.

That has worked in the couple of other cases where this happened.
